Base/Python/slicer/tests/test_slicer_util_load_data.py

```diff
@@ -0,0 +1,33 @@
+import tempfile
+import unittest
+from pathlib import Path
+from urllib.parse import urljoin
+
+import slicer
+
+
+class SlicerUtilLoadDataTests(unittest.TestCase):
+
+    def setUp(self):
+
+        pass
+
+    def test_loadFiberBundle(self):
+
+        hash_algorithm = "SHA256"
+        checksum = "e1848e4d0b90d6c7442b2474c513d0464c6f141253c25aee6f5bcda4c828deac"
+        url = urljoin(slicer.util.TESTING_DATA_URL, hash_algorithm, checksum)
+
+        with tempfile.TemporaryDirectory() as tmpdirname:
+          file_basename = "tractography.vtk"
+          targetFilePath = Path(tmpdirname).joinpath(file_basename)
+          reDownloadIfChecksumInvalid = True
+          slicer.util.downloadFile(url, targetFilePath, checksum=checksum, reDownloadIfChecksumInvalid=reDownloadIfChecksumInvalid)
+
+          returnNode = False
+          node = slicer.util.loadFiberBundle(filename, returnNode)
```

Comment:
`returnNode` is deprecated and `False` is the default value, so it is better not to explicitly add it.

```suggestion
          node = slicer.util.loadFiberBundle(targetFilePath)
```

Reply:
Changed in 9eabaf7.